fe/fe-core/src/main/java/com/starrocks/load/Load.java

```diff
@@ -908,7 +908,6 @@ public static void replayClearRollupInfo(ReplicaPersistInfo info, GlobalStateMgr
             OlapTable olapTable = (OlapTable) db.getTable(info.getTableId());
             Partition partition = olapTable.getPartition(info.getPartitionId());
             MaterializedIndex index = partition.getIndex(info.getIndexId());
-            index.clearRollupIndexInfo();
```

Comment:
This function can be removed.

Reply:
It has been removed.

Reply:
I mean `replayClearRollupInfo` can be removed.

Reply:
Removed, PTAL